pkg/testcore/runner/perf-runner.go

```diff
@@ -213,7 +213,7 @@ func ExecuteSuite(iterCtx context.Context, num int, suites map[string][]suites.I
 
 	if sr.IsStopped() {
 		log.Debug("Suite range stopped")
-		err = fmt.Errorf("suite range stopped")
+		_ = fmt.Errorf("suite range stopped")
```

Comment:
Could you verify if this line is necessary? It seems like we can remove it.

Reply:
This line is not necessary, as the `err` is not being logged or used. It would be best to remove this custom error.